app/css/Mindfulness.css

```diff
@@ -1,28 +1,45 @@
+
 /* Mindfulness.css */
-.video-container {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+/* Set a responsive base font size */
+html {
+  font-size: calc(1rem + 0.5vw); /* Base font size adjusts according to screen size and system settings */
+}
+
+body {
+  font-size: 1rem; /* Base font size for body text */
+}
+
+h1 {
+  font-size: 2rem; /* Scales with the base font size */
 }
 
-.video-wrapper {
-  position: relative;
-  width: 100%;
-  height: 0;
-  padding-bottom: 56.25%; /* 16:9 aspect ratio */
```

Comment:
Why removing this class?

Reply:
This has been a mistake from my side, let me fix this!

Reply:
resolved